libraries/AP_HAL_ChibiOS/hwdef/PixC4-Jetson/hwdef.dat

```diff
@@ -0,0 +1,326 @@
+# HW Definition for Horizon31 PixC4-Jetson
+# The PixC4-Jetson combines an FMUv5 with an Nvidia Jetson companion computer
+# STM32H743 with STM32F103 IOMCU
+# Horizon31, LLC
+
+# MCU class and specific type
+MCU STM32H7xx STM32H743xx
+
+# crystal frequency
+OSCILLATOR_HZ 16000000
+
+define CONFIG_HAL_BOARD_SUBTYPE HAL_BOARD_SUBTYPE_CHIBIOS_FMUV5
```

Comment:
you should not need these FMUv5 lines, much better to use the IMU, BARO and COMPASS lines for specific sets of sensors to probe